modules/tensor_mechanics/include/materials/ComputeIsotropicElasticityTensor.h

```diff
@@ -38,6 +37,9 @@ class ComputeIsotropicElasticityTensor : public ComputeElasticityTensorBase
 
   /// Individual elasticity tensor
   RankFourTensor _Cijkl;
+
+  /// Flag for variable elasticity isotropic tensors
+  MaterialProperty<bool> & _elasticity_tensor_is_constant;
```

Comment:
I talked with @permcody about the single bool option, and he recommended just using a material property because there isn't a good interface in place.